.gitignore

```diff
@@ -258,4 +258,6 @@ lib/*.js.map
 *.spec.js.map
 out/packages
 #tests/unit/*.js # TODO add these once the unit tests are in TS
-#tests/unit/*.js.map
\ No newline at end of file
+#tests/unit/*.js.map
+
+out
```

Comment:
I'm not really sure why output is being checked in. The source should be checked in, the output should just be published to npm.

Reply:
We should probably restructure the directories:

src - the source folder
lib - commonjs modules
lib-es6 - es6 modules
dist - bundles